flocker/control/test/test_model.py

```diff
@@ -1206,10 +1206,34 @@ def test_nonmanifest_datasets_keys_are_their_ids(self):
 
     def test_all_datasets(self):
```

Comment:
The fact it omits replicas is a temporary expedient, not a goal. I suggest having separate test for that which can then be deleted later on.